estate/models/estate_offer.py

```diff
@@ -0,0 +1,29 @@
+from odoo import fields, models, api
+from odoo.tools import relativedelta
+
+class OfferEstate(models.Model):
+    _name = "estate.property.offer"
+    _description = "An offer for a property"
+
+    price = fields.Float("Price")
+    status = fields.Selection(string="Status", copy=False, selection=[
+        ('accepted', 'Accepted'),
+        ('refused', 'Refused'),
+    ])
+
+    partner_id = fields.Many2one("res.partner", string="Partner", required=True)
+    property_id = fields.Many2one("estate.property", string="Property", required=True)
+
+    create_date = fields.Date("Creation Date", default=fields.Date.today)
```

Comment:
This fields already exists because you inherit from `models.Model`, it is redundant to have it here again